steps/waldo-upload/2.0.0/step.yml

```diff
@@ -0,0 +1,99 @@
+title: Waldo Upload
+summary: |
+  Uploads your build to Waldo.
+description: |
+  Uploads your build (`.app`, `.ipa`, or `.apk`) to Waldo.
+
+  You must specify at least a build path and a Waldo upload token.
+
+  Typically the build path is generated by a previous build Step in your
+  Workflow. For example:
+
+  - the **Xcode build for simulator** Step generates a `.app` at `$BITRISE_APP_DIR_PATH`
+  - the **Xcode Archive & Export for iOS** Step generates a `.ipa` at `$BITRISE_IPA_PATH`
+  - the **Android Build** Step generates a `.apk` at `$BITRISE_APK_PATH`
+
+  Each application registered in Waldo is associated with a unique upload
+  token. You can find the upload token for your app by going to the
+  Documentation screen for your app in [Waldo](https://app.waldo.io/). This
+  token _should_ be specified as a _secret_ environment variable.
+website: https://github.com/waldoapp/bitrise-step-waldo-upload
+source_code_url: https://github.com/waldoapp/bitrise-step-waldo-upload
+support_url: https://github.com/waldoapp/bitrise-step-waldo-upload/issues
+published_at: 2021-11-08T10:18:30.95531-05:00
+source:
+  git: https://github.com/waldoapp/bitrise-step-waldo-upload.git
+  commit: 7aca3fe5c5ff75f75aed4d0f81772f1c9a57e33b
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/waldoapp/bitrise-step-waldo-upload
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- build_path: null
+  opts:
+    description: |
+      A `.app`, `.ipa`, or `.apk` path.
+
+      Typically this path is generated by a previous build Step in your
+      Workflow. For example:
+
+      - the **Xcode build for simulator** Step generates a `.app` at `$BITRISE_APP_DIR_PATH`
+      - the **Xcode Archive & Export for iOS** Step generates a `.ipa` at `$BITRISE_IPA_PATH`
+      - the **Android Build** Step generates a `.apk` at `$BITRISE_APK_PATH`
+    is_required: true
+    summary: A `.app`, `.ipa`, or `.apk` path.
+    title: Build path
+- opts:
+    description: |
```

Comment:
Please add *summary* for all Inputs.
You can test the appearance running `$ bitrise :workflow-editor` next to the *bitrise.yml*.